resource/schema/userdata.sql

```diff
@@ -238,11 +257,11 @@ CREATE INDEX deletedItems_dateDeleted ON deletedItems(dateDeleted);
 
 CREATE TABLE libraries (
     libraryID INTEGER PRIMARY KEY,
-    libraryType TEXT NOT NULL,
-    editable INT NOT NULL,
-    filesEditable INT NOT NULL,
+    type TEXT NOT NULL,
+    editable INT NOT NULL DEFAULT 0,
+    filesEditable INT NOT NULL DEFAULT 0,
```

Comment:
I left defaults out for these on purpose because the code creating libraries really does need to make the decision as to editability. Creating a library isn't a common action, so there's no need for this to be implicit.